tests/indent-alignment.test.js

```diff
@@ -1034,6 +1034,31 @@ describe('Indent Alignment', () => {
       ]);
    });
 
+   it('reports no errors for multi-lined html tags', async () => {
+      await testValidExample([
+         '<a class="this is a long attribute which needs to be wrapped" ',
+         'href="#">This query</a>',
+         'testing',
+      ]);
+   });
+
+   it('reports no errors for separately indented blocks', async () => {
+      await testValidExample([
+         '> test',
+         '>',
+         '>                      * test',
+         '>',
+         '>             1. test',
+      ]);
+      await testValidExample([
+         'test',
+         '',
+         '                      * test',
+         '',
+         '             1. test',
+      ]);
```

Comment:
It's a little weird to call these tests "valid" since the list indentation is very weird. Would it make more sense to have more "normal" test cases?

```suggestion
      await testValidExample([
         '> text',
         '>',
         '>    * text',
         '>',
         '>    1. text',
      ]);
      await testValidExample([
         '> text',
         '>',
         '>    1. text',
         '>',
         '>    * text',
      ]);
      await testValidExample([
         '> text',
         '>',
         '>    * text',
         '>',
         '>> text',
         '>>',
         '>>    * text',
      ]);
      await testValidExample([
         '> text',
         '>',
         '>    1. text',
         '>',
         '>> text',
         '>>',
         '>>    1. text',
      ]);
      await testValidExample([
         '> text',
         '>',
         '>    * text',
         '>',
         '>> text',
         '>>',
         '>>    1. text',
      ]);
      await testValidExample([
         '> text',
         '>',
         '>    1. text',
         '>',
         '>> text',
         '>>',
         '>>    * text',
      ]);
```